### PR TITLE
Remove Lucy Parsons Labs references as well as some Chicago specific stuff

### LINKED ISSUE #2: Customize landing page and about pages
Pretty much lucyparsons/OpenOversight#784 - there's a bunch of stuff that points to LucyParsons that we should probably point to our own fork's information (and our twitter, etc.). A good example of what to change is the difference between openoversight.com and cops.photo.

### PR DESCRIPTION
**Pull request by sarayourfriend**
## Status

Ready for review

## Description of Changes

Fixes #2 

Changes proposed in this pull request:

 - Remove references to Lucy Parsons Labs
 - Add a CSS utility to warn us when external links don't have `rel="noopener noreferrer"`
 - Remove some Chicago specific stuff

## Tests and linting

 - [x] I have rebased my changes on current `develop`

 - [ ] pytests pass in the development environment on my local machine

 - [ ] `flake8` checks pass

### CODE REVIEW

**Comment by sarayourfriend** [comment]
It'd be nice if we could get someone to write up some Seattle specific information. Do we know anyone who'd be able to do that?

File: OpenOversight/app/templates/about.html [hunk 1]
```diff
@@ -1,150 +1,158 @@
...
+        <a
+          href="https://lucyparsonslabs.com"
+          rel="noopener noreferrer"
+          target="_blank"
+          >Lucy Parsons Labs</a
+        >
+        project that aims to improve law enforcement visibility and transparency
+        using public and crowdsourced data. We maintain databases, digital
+        galleries, and profiles of individual law enforcement officers from
+        departments across the United States that consolidate information
```
Comment (AetherUnbound) [inline OpenOversight/app/templates/about.html:33]: Did we want to change this to match the meta tag above?

File: OpenOversight/app/templates/about.html [hunk 2]
```diff
@@ -1,150 +1,158 @@
...
+          It is the first project of its kind in the United States, and was
+          first implemented in Chicago in October 2016. OpenOversight launched
+          in the East Bay of the San Francisco Bay Area in fall 2017 and in New
+          York City in 2018. A Baltimore instance was launched in 2019 at
+          <a
+            href="https://bpdwatch.com"
+            rel="noopener noreferrer"
+            target="_blank"
+            >BPDWatch.com</a
+          >.
```
Comment (AetherUnbound) [inline OpenOversight/app/templates/about.html:83]: We could probably mention the Portland instance (cops.photo) launched around the same time too

File: OpenOversight/app/templates/index.html [hunk 3]
```diff
@@ -66,14 +66,11 @@ <h1>
                 <small>Donate</small>
             </h1>
             <p>
-                <!--TODO check on link + copy-->
-                We are entirely volunteer funded and directed. Donations are greatly appreciated, and can be funded
-                through PayPal, YouCaring, or Bitcoin.
-                <br>
-                Note: We are an IRS 501(c)3 not-for-profit and all donations are tax deductible!
+               While Seattle Tech Bloc does not accept donations at this time, if you'd like to financially
+               contirbute to police accountability work in the Seattle area, please donate to Divest SPD.
```
Comment (AetherUnbound) [inline OpenOversight/app/templates/index.html:70]: :heart:

File: OpenOversight/app/templates/submit_image.html [hunk 4]
```diff
@@ -18,7 +18,7 @@ <h1>Submit images <small>of officers in uniform. Officer faces should be clearly
 
    <div class="header">
       <h3>What happens when I submit a photograph?</h3>
-      <p>The next step after a photograph of an officer has been submitted is to match it to the correct badge number, name, and OpenOversight ID (a unique identifier in our system). Volunteers <a href="https://openoversight.com/label">sort through submitted images</a> by first confirming that officers are present in each photograph, and then by matching each photograph to the officer it depicts.</p>
+      <p>The next step after a photograph of an officer has been submitted is to match it to the correct badge number, name, and OpenOversight ID (a unique identifier in our system). Volunteers <a href="{{ url_for('main.get_started_labeling') }}">sort through submitted images</a> by first confirming that officers are present in each photograph, and then by matching each photograph to the officer it depicts.</p>
```
Comment (AetherUnbound) [inline OpenOversight/app/templates/submit_image.html:21]: :100:

File: OpenOversight/app/templates/submit_image.html [hunk 5]
```diff
@@ -84,7 +84,7 @@ <h3>Drop images here to submit photos of officers:</h3>
 
 <div class="container">
    <h3>High Security Submissions</h3>
-   <p>We do not log unique identifying information of visitors to our website, but if you have privacy concerns in submitting photographs, we recommend using <a href="https://www.torproject.org/projects/torbrowser.html.en">Tor Browser.</a></p>
+   <p>We do not log unique identifying information of visitors to our website, but if you have privacy concerns in submitting photographs, we recommend using <a href="https://www.torproject.org/projects/torbrowser.html.en" rel="noopener noreferrer" target="_blank">Tor Browser.</a></p>
```
Comment (AetherUnbound) [inline OpenOversight/app/templates/submit_image.html:87]: It might be good to add a note here about how we scrub EXIF data!

**Review by AetherUnbound — COMMENTED** [review]
Just a couple comments, this is some amazing stuff!

File: OpenOversight/app/templates/about.html (hunk 2 above)
Comment (sarayourfriend) [reply OpenOversight/app/templates/about.html]: Good idea! Done in 1a94762

File: OpenOversight/app/templates/submit_image.html (hunk 5 above)
Comment (sarayourfriend) [reply OpenOversight/app/templates/submit_image.html]: Yup! Done in 6746aa2

File: OpenOversight/app/templates/about.html (hunk 1 above)
Comment (sarayourfriend) [reply OpenOversight/app/templates/about.html]: Oh definitely. Done in 4a4f498

File: OpenOversight/app/templates/about.html [hunk 6]
```diff
@@ -20,19 +20,9 @@
...
-          target="_blank"
-          >Lucy Parsons Labs</a
-        >
-        project that aims to improve law enforcement visibility and transparency
-        using public and crowdsourced data. We maintain databases, digital
-        galleries, and profiles of individual law enforcement officers from
-        departments across the United States that consolidate information
-        including names, birthdates, mentions in news articles, salaries, and
-        photographs.
+        OpenOversight is a <a href="" target="_blank" rel="noopener noreferrer">Seattle Tech Bloc</a> project that aims to improve law enforcement
```
Comment (AetherUnbound) [inline OpenOversight/app/templates/about.html:23]: Was this supposed to link to our twitter?

**Review by AetherUnbound — COMMENTED** [review]
> We should also ask the folks at DivestSPD to give us a blurb about "next steps" after identifying an officer. Right now the next steps are Chicago PD specific:

Great call, I'll make an issue for it

File: OpenOversight/app/templates/about.html (hunk 6 above)
Comment (sarayourfriend) [reply OpenOversight/app/templates/about.html:23]: Whoops yes!

**Review by AetherUnbound — APPROVED** [review]
:rocket:

**Pull request merged**